src/pages/developers/architecture/protocol/contracts/Revert.sol/interface.Revertable.md

```diff
@@ -1,5 +1,5 @@
 # Revertable
-[Git Source](https://github.com/zeta-chain/protocol-contracts/blob/3a274ce7bad045a879c73669586611d35509cbce/contracts/Revert.sol)
+[Git Source](https://github.com/zeta-chain/protocol-contracts/blob/dedf2ca4d335fe85937fd686450fecebb5456bc9/contracts/Revert.sol)
```

Comment:
@fadeev can't we make this point to the main branch so we don't need to track these hashes?

Reply:
https://github.com/zeta-chain/protocol-contracts/pull/392